### ISSUE TITLE
Edit uncomment file

### ISSUE THREAD

**Issue by zippaaa** [ForkIssues/TrackerWin#356]
Please add the ability to edit the file until it is uncommitted.
![shot_190710_151740](https://user-images.githubusercontent.com/22454101/60970248-3c789480-a32a-11e9-8f4e-a933351b7d5d.png)
(Not stage/discard line)

**Comment by jerone** [comment]
You mean like an editor?

**Comment by zippaaa** [comment]
Yes. I check uncommitted files and want to correct one of them without exit from the Fork.
The Fork has two buttons ("stage line", "discard line") and If it'd has the opportunity to edit the text. It'd be great.

Also, when I look diffs between last commit of current branch and any other commit, I'd want to edit files of **current branch** .
![shot_190711_125956](https://user-images.githubusercontent.com/22454101/61046611-48299100-a3e6-11e9-820f-9d9d647390fe.png)
